pkg/quotaplugins/quota-forest/quota-manager/quota/quotamanager.go

```diff
@@ -369,33 +369,40 @@ func (m *Manager) UpdateForest(forestName string) (unallocatedConsumerIDs []stri
 // AllocateForest : allocate a consumer on a forest
 func (m *Manager) AllocateForest(forestName string, consumerID string) (response *core.AllocationResponse, err error) {
 	forestController := m.forests[forestName]
+	var awExistsInNamespace bool = false
```

Comment:
Move this closer to where it is used. I'd say line 392

Reply:
Done